lib/dl_configs/dl_configs/settings_loaders/loader_env.py

```diff
@@ -72,9 +70,10 @@ def field_set(self) -> set[str]:
         return self._field_set
 
 
-def get_sub_keys(prefix: str, env: SDict):  # type: ignore  # 2024-01-24 # TODO: Function is missing a return type annotation  [no-untyped-def]
+def get_sub_keys(prefix: str,
+                 env: SDict):  # type: ignore  # 2024-01-24 # TODO: Function is missing a return type annotation  [no-untyped-def]
```

Comment:
Are all these new-lines really necessary? I'm feeling a hint of formatter misconfiguration